src/http-api/resources/version.js

```diff
@@ -0,0 +1,14 @@
+var ipfs = require('./../index.js').ipfs
+var boom = require('boom')
+
+exports = module.exports
```

Comment:
because although they point to the same thing initially, if not binded, once exports changes it will stop pointing to module.exports and therefore none of the functions that gets exported by `exports.` will be exposed

more info and examples:
https://blog.tableflip.io/the-difference-between-module-exports-and-exports/